eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -703,4 +704,55 @@ private function addUrlWildcardsSection($rootNode): ArrayNodeDefinition
                 ->end()
             ->end();
     }
+
+    /**
+     * Defines configuration for Doctrine ORM.
+     *
+     * The configuration is available at:
+     * <code>
+     * ezpublish:
```

Comment:
3.x note: needs to be changed on merge to `ezplatform` as this is the one officially supported (possible for 2.5 but with 2.5 optional EzPlatformCoreBundle)